src/components/error/Error.tsx

```diff
@@ -0,0 +1,38 @@
+import './Error.css'
+import React from 'react'
+import spilledTea from "../../images/Coffee-Burst.svg";
+import { useEffect  } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+interface Props {
+    isRedirected: boolean | any;
+    setIsRedirected: React.Dispatch<React.SetStateAction<boolean | undefined>>
+}
+
+const Error: React.FC<Props> = ({ isRedirected, setIsRedirected }) => {
+    // let location = useLocation()
+    // console.log(location)
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        setIsRedirected(true)
+    }, [setIsRedirected])
+
+    useEffect(() => {
+       window.onbeforeunload = () => {
+        //    setIsRedirected(false)
+        if(isRedirected === true) {
+            navigate('../', {replace: true})
+        }
+       }
+    }, [isRedirected, navigate, setIsRedirected])
+
+    return (
+        <div className='error_wrapper'>
```

Comment:
Might be a good time to try using a [fragment](https://legacy.reactjs.org/docs/fragments.html).